src/Components/Web.JS/src/Services/ComponentDescriptorDiscovery.ts

```diff
@@ -149,7 +149,8 @@ function getComponentComment(commentNodeIterator: ComponentCommentIterator, type
             return createServerComponentComment(componentComment as ServerComponentComment, candidateStart, commentNodeIterator);
         }
       } catch (error) {
-        throw new Error(`Found malformed component comment at ${candidateStart.textContent}`);
+        // throw new Error(`Found malformed component comment at ${candidateStart.textContent}`);
+        // FIXME: Temporary hack until we know what to do about the duplicate, malformed comments.
```

Comment:
I noticed that, for every valid component comment, there is also a component comment with the same `prerenderId` but no other information, e.g.: `<!--Blazor:{"prerenderId":"825916660abd46248dd9d28f894e203b"}-->`. This causes this method to fail. @SteveSandersonMS is this behavior expected?

Reply:
Resolving because this was discussed offline.